Makefile

```diff
@@ -46,14 +46,21 @@ endif
 	bandit -r $(checkfiles)
 	python setup.py check -mrs
 
-test: deps
-	$(py_warn) TORTOISE_TEST_DB=sqlite://:memory: py.test
+test: test_sqlite
 
-_testall:
+test_sqlite: deps
 	$(py_warn) TORTOISE_TEST_DB=sqlite://:memory: py.test --cov-report=
-	python -V | grep PyPy || $(py_warn) TORTOISE_TEST_DB=postgres://postgres:$(TORTOISE_POSTGRES_PASS)@127.0.0.1:5432/test_\{\} py.test --cov-append --cov-report=
```

Comment:
Step 1: Don't make anyting under `_testall` depend on `deps`. The reason is that pypy doesn't support some C plugins, like `asyncpg`.
Step 2: add the `python -V | grep PyPy || ` in front of the postgres test, that disables the test for PyPy, but allows it for CPython.
Step 3: ???
Step 4: Profit!